nodes/src/nodes/authoritative_overlay/README.md

```
@@ -0,0 +1,25 @@
+# Authoritative Overlay Node
+
+Part of the **Audit-grade financial extraction node suite**.
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)